openassessment/tests/test_data.py

```diff
@@ -579,13 +580,15 @@ def test_collect_ora2_responses(self):
             COURSE_ID,
             team_item_id,
             'team_1',
-            student_model_ids[:3]
+            student_model_1.id,
```

Comment:
This feels like instead of artificially constraining these lists we should have factories and test setup that establishes lists for team memberships.

Not technically in scope for this PR but a place to consider refactoring, if not now then in the future.

Reply:
tbh not totally sure what you mean, but I got rid of the slicing